package-lock.json

```diff
@@ -28,6 +28,107 @@
         "loglevel": "^1.6.1",
         "make-fetch-happen": "^8.0.0",
         "pretty-format": "^26.0.0"
+      },
+      "dependencies": {
+        "apollo-cache-control": {
+          "version": "0.11.4",
+          "resolved": "https://registry.npmjs.org/apollo-cache-control/-/apollo-cache-control-0.11.4.tgz",
+          "integrity": "sha512-FUKE8ASr8GxVq5rmky/tY8bsf++cleGT591lfLiqnPsP1fo3kAfgRfWA2QRHTCKFNlQxzUhVOEDv+PaysqiOjw==",
+          "requires": {
+            "apollo-server-env": "^2.4.5",
```

Comment:
The entirety of this `package-lock.json` change seems unexpected.

Reply:
6937cd8